actions/code.js

```diff
@@ -18,20 +18,30 @@ function wait(timeout) {
 // eslint-disable-next-line consistent-return,func-names
 exports.process = async function (msg, conf, snapshot) {
   const ctx = vm.createContext({
-    _,
+    // Node Globals
+    Buffer,
+    clearInterval,
+    clearTimeout,
     console,
+    exports: {},
+    global: {},
+    module: { exports },
```

Comment:
```suggestion
    module: { exports: {} },
```

I think exports should be the empty object to match line 26

Reply:
Actually, I think it makes sense to declare an empty object beforehand and pass that as both `exports: someEmptyObject` and `module: { exports: someEmptyObject } }` since they should refer to the same memory address?

Reply:
I thought I was doing that but apparently I am not...